nmtwizard/preprocess/operators/noise.py

```diff
@@ -96,10 +99,14 @@ def _preprocess_tu(self, tu, *args):
         src_tok = tu.src_tok
         tokens = src_tok.token_objects[0]
         new_tokens = self._apply_word_noise(tokens)
+        result = [tu]
+        if self._data_augmentation:
```

Comment:
I'm thinking about this now: should we check the TU was actually modified before returning the original version? If no noise are applied, the same TU is duplicated in the current code.

Reply:
Yes, you are right, as usual :) I'll make another PR for this fix.